pkg/services/certgenerator/certutil.go

```diff
@@ -199,7 +206,8 @@ func (cu *CertUtil) InitializeCACertPKI() error {
 	exists, err := certutil.CanReadCertAndKey(cu.CACertFile(), cu.CAKeyFile())
 
 	if err != nil {
-		return fmt.Errorf("error reading existing CA PKI: %s", err.Error())
+		return fmt.Errorf("error reading existing CA PKI: %s"+
+			CannotVerifyErrorMessageFragment, err.Error())
```

Comment:
It's good practice to keep format strings as close to string literals as possible, concatenating like this you would run into issues if for some reason `CannotVerifyErrorMessageFragment` had a sprintf placeholder in it.
```suggestion
		return fmt.Errorf("error reading existing CA PKI: %s %s",
			err.Error(), CannotVerifyErrorMessageFragment)
```